libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -702,29 +713,39 @@ void AP_GPS::detect_instance(uint8_t instance)
             new_gps = new AP_GPS_UBLOX(*this, state[instance], _port[instance], role);
         }
 #ifndef HAL_BUILD_AP_PERIPH
+#if AP_GPS_SBP2_ENABLED
```

Comment:
We should turn AP_GPS_SBP2_ENABLED off in chibios_hwdef.dat in the "add defaults for periph" block.  Maybe a future PR?